maps/include/maps/FlatSkyMap.h

```diff
@@ -55,7 +60,7 @@ class FlatSkyMap : public G3FrameObject, public G3SkyMap {
 	double &operator [] (size_t i) override;
 
 	double at(size_t x, size_t y) const;
-	double operator () (size_t x, size_t y) const { return this->at(x, y); };
+	double operator () (size_t x, size_t y) const {	return this->at(x, y); };
```

Comment:
What's going on here? Did you replace a space with a space?

Reply:
no clue.  it may have accidentally been a tab?